### PR TITLE
feat: save relativeFrom and absoluteFrom in info object, #538

### PR DESCRIPTION
**Pull request by madflow**
This PR contains a:

- [ ] **bugfix**
- [x] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

See #538 

### Breaking Changes

None I would know of

### Additional Info

None

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -471,7 +472,7 @@ class CopyPlugin {
                 `writing "${webpackTo}" to compilation assets from "${absoluteFrom}"`
               );
 
-              const info = { copied: true };
+              const info = { copied: true, absoluteFrom, relativeFrom };
```
Comment (evilebottnawi) [inline src/index.js:475]: No need `absoluteFrom` and `relativeFrom`, we need only `originalFilename`

Comment (evilebottnawi) [reply src/index.js:475]: `absoluteFrom` do you stats are unstable on different platforms

Comment (madflow) [reply src/index.js:475]: What would qualify as `originalFilename` ? I would argue that this is `absoluteFrom`... The asset object has the following fields:

```
        absoluteFrom: '/home/flo/Projects/Javascript/copy-webpack-plugin/test/fixtures/directory/nested/nestedfile.txt',
        relativeFrom: 'nestedfile.txt',
        webpackTo: 'nested/nestedfile.txt',
        data: <Buffer >,
        targetPath: 'nested/nestedfile.txt',
        force: undefined
```

**Review by evilebottnawi — CHANGES_REQUESTED** [review]
Not resolved

**Comment by madflow** [comment]
> Not resolved

Sorry - clicked by accident I guess.

File: src/index.js (hunk 1 above)
Comment (evilebottnawi) [reply src/index.js:475]: Original filename - relative path where file was readed

Comment (madflow) [reply src/index.js:475]: Given
```
├── css
│   ├── customera
│   │   └── styles.css
│   └── customerb
│       └── styles.css
└── webpack.config.js
```

I would need `css/customera/styles.css` and `css/customerb/styles.css` as documented in the info object. Is that would you mean?

Comment (evilebottnawi) [reply src/index.js:475]: Based on your `context`

Comment (evilebottnawi) [reply src/index.js:475]: Default contest is process.cwd(), so use you have `css/customera/styles.css`

**Comment by madflow** [comment]
This PR was done pretty prematurely. I will close this for now - looking into the proposed changes for a potential follow up.

**Pull request closed without merging**